rules/coding-style/src/Rector/Class_/AddArrayDefaultToArrayPropertyRector.php

```diff
@@ -176,8 +176,6 @@ private function clearNotNullBeforeCount(Class_ $class, array $propertyNames): v
             if (! $node instanceof BooleanAnd) {
                 return null;
             }
-
-            // $this->value !== null
```

Comment:
Were this comments deleted manually or should the rector preserve them?

Reply:
it seems by rectify command that delete commented code

Reply:
@samsonasik Could you return manually those comments? 

It's out of scope this pull-request, so just to preserve them here

Reply:
reverted manually.